shape-calculator/src/components/ShapeForm.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+
+const ShapeForm = (props) => {
+    const handleSetShape = (e) => {
```

Comment:
You can combine these function calls all into one maybe pass a parameter to figure out which is being set